aten/src/ATen/native/cudnn/RNN.cpp

```diff
@@ -1219,7 +1219,7 @@ Tensor try_get_weight_buf(
   return weight_buf;
 }
 
-const char * WEIGHT_FORMAT_WARN = "RNN module weights are not part of single contiguous "
+const char * WEIGHT_FORMTORCH_WARN = "RNN module weights are not part of single contiguous "
```

Comment:
I think this is a spew from the replace ;)

Reply:
Good catch!